client/lib/abtest/active-tests.js

```diff
@@ -45,6 +45,14 @@ module.exports = {
 		},
 		defaultVariation: 'original'
 	},
+	sidebarPlanLink: {
+		datestamp: '20160101',
+		variations: {
+			plans: 50,
+			'plans/my-plan': 50,
+		},
+		defaultVariation: 'plans'
```

Comment:
I think it's ok to run this just for new users. I would set up the test to determine impact on new-user-one-month-refunds. So we might as well focus it on them anyway.